build/build.js

```diff
@@ -168,7 +168,8 @@ var preprocess = function (dependencies) {
         else 
             src = pp.process({
                 PROFILER: profiler || debug,
-                DEBUG: debug
+                DEBUG: debug,
+                RELEASE_BUILD: compilerLevel != COMPILER_LEVEL[0]
```

Comment:
Any reason not to have this as `RELEASE` instead of `RELEASE_BUILD'?

Reply:
And just to be clear, we have the following build configs:

* DEBUG: playcanvas.dbg.js
* PROFILER: playcanvas.prf.js
* RELEASE: playcanvas.js
* RELEASE (MINIFIED): playcanvas.min.js

So MiniStats is just in the first two?

Reply:
I should probably have, but at the time RELEASE just seemed too general.

Reply:
MiniStats is available in all but the last RELEASE(MINIFIED) since this is the only build which specifies compilerLevel.

Reply:
I get your point, but the same would hold for `DEBUG`. I think we should ensure consistency of naming here. And I guess I'm fine with `playcanvas.js` containing MiniStats. We can always modify this decision later if required.

Reply:
Just thinking about it....so if `playcanvas.js` isn't `DEBUG`, `PROFILER` or `RELEASE`....what is it? 😄

Reply:
I mean, `playcanvas.js` and `playcanvas.min.js` are functionally identical (and so they should be). You should have near 100% confidence that your game will behave the same on both. I'm a little bit worried about diverging these two library flavors.

Reply:
Renamed RELEASE_BUILD -> RELEASE.